app/controllers/me/index.js

```diff
@@ -1,20 +1,20 @@
 import Ember from 'ember';
-import ajax from 'ic-ajax';
+
+const { inject: { service } } = Ember;
```

Comment:
this will make it harder for the new modules codemod to transform it into an import from `@ember/service`... I'd rather just keep using the globals for now.